pkg/commands/options/build.go

```diff
@@ -76,7 +76,7 @@ func AddBuildOptions(cmd *cobra.Command, bo *BuildOptions) {
 		"Disable optimizations when building Go code. Useful when you want to interactively debug the created container.")
 	cmd.Flags().StringVar(&bo.SBOM, "sbom", "spdx",
 		"The SBOM media type to use (none will disable SBOM synthesis and upload, also supports: spdx, go.version-m).")
-	cmd.Flags().StringVar(&bo.Platform, "platform", "",
+	cmd.Flags().StringSliceVar(&bo.Platforms, "platform", []string{},
 		"Which platform to use when pulling a multi-platform base. Format: all | <os>[/<arch>[/<variant>]][,platform]*")
```

Comment:
The format here should be updated, to indicate that the value is no longer a comma-separated string.

Reply:
comma-separated string is still supported, so existing usage with commas _should_ still work (I tested locally, but I don't think there is bash-level test in the project?)

Reply:
TIL, thanks